src/app.js

```diff
@@ -1,10 +1,21 @@
-const path = require('path');
 const express = require('@feathersjs/express');
+const feathers = require('@feathersjs/feathers');
+const configuration = require('@feathersjs/configuration');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to resolve path to module '@feathersjs/configuration'.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=4605061)